.gitignore

```diff
@@ -28,6 +28,7 @@ dist-ssr
 # firebase
 /.firebase
 /firebase
+/lib/firebase.js
```

Comment:
this shouldn't be in the gitignore as this file has stuffs to be changed by others!